### PR TITLE
Some very basic doom profile troubleshooting

### PR DESCRIPTION
**Pull request by bernokl**

### CODE REVIEW

File: docs/TROUBLESHOOT_DOOM.org [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+echo $HUMACSDIR
+#+END_SRC
+
+* Run build and sync
+This should update and repair any issues with packages
+#+BEGIN_SRC shell :results raw
+$HUMACSDIR/doom-emacs/bin/doom sync
+#+END_SRC
+
+Not 100% on value of building again, but I do not think it hurts anything
```
Comment (zachmandeville) [inline docs/TROUBLESHOOT_DOOM.org:22]: building will "byte-compile and symlink installed packages".  It shouldn't be necessary when running doom sync, and I think this part could be removed.  The heading would then just be 'Run Doom Sync'

File: docs/TROUBLESHOOT_DOOM.org [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+#+BEGIN_SRC shell :results raw
+$HUMACSDIR/doom-emacs/bin/doom build
+#+END_SRC
+
+* Additional steps
+This will give you some additional info if you are having issues
+#+BEGIN_SRC shell :results raw
+emacs -nw --debut-init
```
Comment (zachmandeville) [inline docs/TROUBLESHOOT_DOOM.org:30]: change to `emacs -nw --debug-init`

File: docs/TROUBLESHOOT_DOOM.org [hunk 3]
```diff
@@ -0,0 +1,35 @@
+#+TITLE: Troubleshoot_doom
+Some very basic steps to check if your doom instance is giving you trouble
+
+* Make sure your env vars is set correctly
+- EMACSLOADPATH :: path to humacs repo, ending with : (e.g. ~/ii/humacs:)
+- DOOMDIR :: path to doom-config in the humacs repo (e.g. ~/ii/humacs/doom-config)
+- HUMACS_PROFILE :: e.g. doom
+- HUMACSDIR :: Easy path to dir, not required but handy
```
Comment (zachmandeville) [inline docs/TROUBLESHOOT_DOOM.org:8]: keeping with the style of definitions above, perhaps add (e.g `~/ii/humacs`)

**Review by zachmandeville — CHANGES_REQUESTED** [review]
Looks good!  There are just some tiny changes that will make it 100%!

File: docs/TROUBLESHOOT_DOOM.org (hunk 1 above)
Comment (bernokl) [reply docs/TROUBLESHOOT_DOOM.org]: removed entire doom build secion, changed heading to "Run Doom sync"

File: docs/TROUBLESHOOT_DOOM.org (hunk 2 above)
Comment (bernokl) [reply docs/TROUBLESHOOT_DOOM.org]: fixed typo, ty

File: docs/TROUBLESHOOT_DOOM.org (hunk 3 above)
Comment (bernokl) [reply docs/TROUBLESHOOT_DOOM.org]: added "eg..."

**Pull request merged**